examples/opensora_hpcai/configs/opensora-v1-1/inference/sample_t2v.yaml

```diff
@@ -0,0 +1,25 @@
+ckpt_path: "models/opensora_v1.1_stage3.ckpt"
+t5_model_dir: "models/t5-v1_1-xxl/"
+vae_checkpoint: "models/sd-vae-ft-ema.ckpt"
+
+image_size: [ 640, 360 ]
+num_frames: 16
+frame_interval: 3
+fps: 24
+enable_flash_attention: True
+model_max_length: 200
+dtype: "fp32"
+batch_size: 1
+
+# sampling
+sampling_steps: 100
+guidance_scale: 7.0
+guidance_channels: 3
+seed: 42
+ddim_sampling: False
+
+loop: 2
```

Comment:
I think loop=1 may be bettter for a more standard i2v task unless users know that they want to loop from prveious frame

Reply:
done